### DISCUSSION TITLE
Can ASSIMP export FBX with metadata in aiNode?

### DISCUSSION THREAD

**Discussion by fuhaixi** [assimp/assimp#5670] [Q&A] [answered]
I am trying to export a aiscene with some ainode that own metadata to fbx. The Assimp can read fbx with metadata. So can it export fbx with metadata too?

**Comment by kimkulling** [comment, marked as answer]
This question is hard to answer: the metadata are FBX-related data mapped to meta-data attributes. So most data will be exported. 

If you find any gap just let us know. Then we can fix that in the exporter.
